src/browser/karma.conf.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(config) {
       // suppress annoying 404 warnings for resources, images, etc.
       { pattern: 'dist/dev/assets/**/*', watched: false, included: false, served: true },
 
-      'test-main.js'
+      'src/browser/test-main.js'
```

Comment:
Should not be copied because it is not used there. We should use the version in `src/browser`.